core/lb_test.go

```diff
@@ -107,7 +144,8 @@ func TestLB_SessionCancel(t *testing.T) {
 
 	sess := &transcoderSession{
 		transcoder:  newStubTranscoder(""),
-		sender:      make(chan *transcoderParams, 1),
+		done:        make(chan struct{}, 1),
```

Comment:
Not that it matters in the context of this test, but this can be unbuffered as is the case in the actual LoadBalancingTranscoder impl right?